src/main/java/com/atlauncher/managers/InstanceManager.java

```diff
@@ -102,39 +85,39 @@ public static void loadInstances() {
                 }
 
                 if (instance.launcher.curseForgeManifest != null
-                        && instance.launcher.curseForgeManifest.projectID != null
-                        && instance.launcher.curseForgeManifest.fileID != null) {
+                    && instance.launcher.curseForgeManifest.projectID != null
+                    && instance.launcher.curseForgeManifest.fileID != null) {
                     LogManager.info(String.format("Converting instance \"%s\" CurseForge information",
-                            instance.launcher.name));
+                        instance.launcher.name));
                     instance.launcher.curseForgeProject = CurseForgeApi
-                            .getProjectById(instance.launcher.curseForgeManifest.projectID);
+                        .getProjectById(instance.launcher.curseForgeManifest.projectID);
                     instance.launcher.curseForgeFile = CurseForgeApi.getFileForProject(
-                            instance.launcher.curseForgeManifest.projectID,
-                            instance.launcher.curseForgeManifest.fileID);
+                        instance.launcher.curseForgeManifest.projectID,
+                        instance.launcher.curseForgeManifest.fileID);
                     instance.launcher.curseForgeManifest = null;
 
                     instance.save();
                 }
 
                 if (instance.launcher.numPlays == null) {
                     LogManager.info(String.format("Converting instance \"%s\" numPlays/lastPlayed",
-                            instance.launcher.name));
+                        instance.launcher.name));
                     instance.launcher.numPlays = instance.numPlays;
                     instance.launcher.lastPlayed = instance.lastPlayed;
 
                     instance.save();
                 }
 
                 if (instance.launcher.account != null
-                        && !AccountManager.isAccountByName(instance.launcher.account)) {
```

Comment:
I really need to get some tool in that does consistent formatting on save and commit cross IDEs like Prettier does for JS/TS.

Reply:
InteliJ has it baked in. You need to go to the styles settings panel and apply the style to be project only.

Reply:
Yeah looking for something editor agnostic and can run with git hooks which has been the issue. I'll look into this again